templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -751,18 +749,30 @@ contents:
     # Setup an exit trap to rollback on error
     handle_exit() {
       e=$?
-      [ $e -eq 0 ] && print_state && exit 0
+      tdir=$(mktemp -u -d -t "configure-ovs-$(date +%Y-%m-%d-%H-%M-%S)-XXXXXXXXXX")
+      
+      if [ $e -eq 0 ]; then
+        print_state
+        # remove previous troubleshooting information
+        rm -rf "$(dirname "$tdir")/configure-ovs-*"
+        exit 0
+      fi
 
       echo "ERROR: configure-ovs exited with error: $e"
       print_state
 
-      # copy configuration to tmp
-      dir=$(mktemp -d -t "configure-ovs-$(date +%Y-%m-%d-%H-%M-%S)-XXXXXXXXXX")
+      # remove previous troubleshooting information except the oldest one
+      mapfile -t tdirs < <(compgen -G "$(dirname "$tdir")/configure-ovs-*)")
```

Comment:
This doesnt work for me when I tested it locally.  I create a few dirs named `configure-ovs-*` in tmp, removed dirname part and put in /tmp.

Reply:
Good catch, it should be `mapfile -t tdirs < <(compgen -G "$(dirname "$tdir")/configure-ovs-*")`
Will fix